db/migrate/20190213083129_create_tests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTests < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tests do |t|
+      t.string :email
+      t.string :password
+
+      t.timestamps
+    end
+  end
```

Comment:
Why did you create a migration for this feature?

Reply:
I think this is related the incorrect creation of second db as above